implementation/endpoints/src/netlink_connector.cpp

```diff
@@ -180,9 +189,37 @@ void netlink_connector::receive_cbk(boost::system::error_code const &_error,
                 }
                 case NLMSG_ERROR: {
                     struct nlmsgerr *errmsg = (nlmsgerr *)NLMSG_DATA(nlh);
-                    VSOMEIP_ERROR << "netlink_connector::receive_cbk received "
-                        "error message: " << std::dec << nlh->nlmsg_type
-                        << " seq " << errmsg->msg.nlmsg_seq;
+                    if (errmsg->error == 0) {
+                        // Ack from netlink
+                        break;
+                    }
+
+                    unsigned int retry = errmsg->msg.nlmsg_seq >> retry_bit_shift;
+                    if (retry >= max_retries) {
+                        VSOMEIP_ERROR << "netlink_connector::receive_cbk received "
+                            "error message: " << strerror(-errmsg->error)
+                            << " type " << std::dec << errmsg->msg.nlmsg_type
+                            << " seq " << errmsg->msg.nlmsg_seq;
+                        break;
+                    }
+
+                    unsigned int request_sequence = errmsg->msg.nlmsg_seq & request_sequence_bitmask;
```

Comment:
just to be coherent with the rest of the code.
```suggestion
                   std::uint32_t request_sequence = errmsg->msg.nlmsg_seq & request_sequence_bitmask;
```